modules/dynamic-plugins/ref-community-plugins.adoc

```diff
@@ -10,9 +10,17 @@
 Details on how {company-name} provides support for bundled community dynamic plugins are available on the https://access.redhat.com/policy/developerhub-support-policy[Red Hat Developer Support Policy] page.
 ====
 
-{product-very-short} includes the following 0 community plugins:
+{product-very-short} includes the following 2 community plugins:
 
 [%header,cols=4*]
 |===
 |*Name* |*Plugin* |*Version* |*Path and required variables*
+|Argo CD  |`https://npmjs.com/package/@roadiehq/backstage-plugin-argo-cd/v/2.8.4[@roadiehq/backstage-plugin-argo-cd]` |2.8.4 
+|`./dynamic-plugins/dist/roadiehq-backstage-plugin-argo-cd`
+
+
+|Global Floating Action Button  |`https://npmjs.com/package/@red-hat-developer-hub/backstage-plugin-global-floating-action-button/v/1.0.0[@red-hat-developer-hub/backstage-plugin-global-floating-action-button]` |1.0.0 
```

Comment:
this will switch to GA supported in 1.5.1 - https://issues.redhat.com/browse/RHIDP-6405